wouso/interface/cpanel/urls.py

```diff
@@ -75,7 +75,8 @@
     url(r'^races/all/$', 'wouso.interface.cpanel.views.races_groups', name='races_groups'),
     url(r'^races/add/$', 'wouso.interface.cpanel.views.races_add', name='races_add'),
     url(r'^races/groups/add/$', 'wouso.interface.cpanel.views.group_add', name='group_add'),
-
+    url(r'^races/toggle/can_play/$', 'wouso.interface.cpanel.views.race_toggle_can_play', name='race_toggle_can_play'),
```

Comment:
@catalin-oancea, the URL is still not appropriate. Rather than requesting data (that's what a GET request mean, after all) from a generic URL, you should send a PUT request to a specific URL, which should be the URL for a specific race.